dbms/src/Interpreters/SystemLog.cpp

```diff
@@ -48,6 +49,14 @@ SystemLogs::SystemLogs(Context & global_context, const Poco::Util::AbstractConfi
     part_log = createSystemLog<PartLog>(global_context, "system", "part_log", config, "part_log");
     trace_log = createSystemLog<TraceLog>(global_context, "system", "trace_log", config, "trace_log");
     text_log = createSystemLog<TextLog>(global_context, "system", "text_log", config, "text_log");
+    metric_log = createSystemLog<MetricLog>(global_context, "system", "metric_log", config, "metric_log");
+
+    if (metric_log)
+    {
+        constexpr size_t DEFAULT_METRIC_LOG_COLLECT_INTERVAL_MILLISECONDS = 1000;
```

Comment:
Let's make no default at all. (config.get... with throw an exception) and keep the value in example in config.